.travis.yml

```diff
@@ -1,34 +0,0 @@
-# Generated from:
-# https://github.com/zopefoundation/meta/tree/master/config/pure-python
-language: python
-python:
```

Comment:
Maybe let's keep it for a bit longer, while we evaluate GitHub actions?  It's not currently broken, is it?

Reply:
I removed it in the PR so not each try to get it running runs on TravisCI using up our credits over there.

Reply:
According to https://travis-ci.com/github/zopefoundation?tab=insights&timeInterval=week we used more than 1,000 minutes last week. So IMHO we should not eat up the Travis credits for repositories migrated to GHA.